native/src/components/OverlayButton.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { ReactElement } from 'react'
+import styled from 'styled-components/native'
+
+import { CloseIcon } from '../assets'
+import Text from './base/Text'
+
+const Container = styled.Pressable`
+  flex: 1;
+  flex-direction: row;
+  height: 30px;
+  padding: 4px 8px;
+  align-items: center;
+  margin: 0 2px;
+  border-radius: 20px;
+  background-color: ${props => props.theme.colors.backgroundColor};
```

Comment:
```suggestion
  background-color: ${props => props.theme.colors.backgroundColor};
  gap: 4px;
```

Reply:
I don't see any difference here/not working for me.

Reply:
<img width="107" alt="image" src="https://github.com/digitalfabrik/integreat-app/assets/18513943/1ae3e8b5-feaf-4882-aca5-7d6cc3e1a488">
This is without a gap.

<img width="115" alt="image" src="https://github.com/digitalfabrik/integreat-app/assets/18513943/613a7f0e-d220-4510-a556-5500a9102678">
This is with a gap.